package.json

```diff
@@ -15,28 +15,37 @@
   "author": "Benjamim Sonntag <benjamimsonntag@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "rm -rf dist && babel src --out-dir dist --ignore \"**/*.test.js\"",
-    "format": "prettier --write \"./**/*.{js,json}\"",
-    "lint": "eslint . && prettier --check \"./**/*.{js,json}\"",
+    "type-check": "tsc --noEmit",
+    "type-check:watch": "npm run type-check -- --watch",
+    "build": "npm run build:types && npm run build:js",
```

Comment:
There is no `build:types` script. Did you forget to add it?

Reply:
Sorry, i meant to run the `type-check` script. Fixed.